packages/shared/src/components/widgets/SimilarPosts.tsx

```diff
@@ -52,58 +53,66 @@ const DefaultListItem = ({
   post,
   onLinkClick,
   onBookmark,
-}: PostProps): ReactElement => (
-  <article
-    className={classNames(
-      'group relative flex items-start py-2 pl-4 pr-2 hover:bg-surface-hover',
-      styles.card,
-    )}
-  >
-    <CardLink
-      href={post.commentsPermalink}
-      title={post.title}
-      {...combinedClicks(() => onLinkClick(post))}
-    />
-    <LazyImage
-      imgSrc={post.source.image}
-      imgAlt={post.source.name}
-      className={imageClassName}
-    />
-    <div className={textContainerClassName}>
-      <h5
-        className={classNames(
-          'multi-truncate mb-0.5 text-ellipsis break-words text-text-primary typo-callout',
-          styles.title,
-        )}
-      >
-        {post.title}
-      </h5>
-      {post.trending ? (
-        <div className="flex items-center text-text-tertiary typo-footnote">
-          <HotLabel />
-          <div className="ml-2">{post.trending} devs read it last hour</div>
-        </div>
-      ) : (
-        <PostEngagementCounts
-          upvotes={post.numUpvotes}
-          comments={post.numComments}
-          className="text-text-tertiary"
-        />
+}: PostProps): ReactElement => {
+  const { showPlusSubscription } = usePlusSubscription();
```

Comment:
In light of the discussion, it seems we don't have to check for Plus now 😅

Reply:
@sshanzel Done, and also removed tests, variables and functions that are no longer used.